packages/fxa-content-server/app/scripts/lib/url.js

```diff
@@ -112,18 +131,16 @@ export default {
    * @returns {String}
    */
   objToUrlString(obj = {}, prefix = '?') {
-    const params = [];
-    // eslint-disable-next-line no-unused-vars
-    for (const paramName in obj) {
-      const paramValue = obj[paramName];
-      if (
-        typeof paramValue !== 'undefined' &&
-        paramValue !== null &&
-        paramValue !== ''
-      ) {
-        params.push(paramName + '=' + encodeURIComponent(paramValue));
-      }
-    }
+    const params = this._getObjPairs(obj)
+      .map(([[key0, ...keysRest], value]) => {
+        value = value.toString();
+        if (value.length) {
+          return `${key0}${keysRest
+            .map((a) => `[${a}]`)
+            .join('')}=${encodeURIComponent(value)}`;
```

Comment:
I can't find any good authoritative SurveyGizmo documentation on how to pass a list of values, but from the comment section of an SurveyGizmo, I think it wants a comma separate values string.  If that's the case, this won't work.

Maybe @johngruen can help on the format question here.

Reply:
@chenba [this article](https://help.surveygizmo.com/help/url-variables) appears to indicate that regular query params are accepted. Or are you referring to something else?

Reply:
Oh are you talking about an array value? Yeah, I'm not sure either. I would prefer to keep it like this, but if SG doesn't like it I suppose I could modify the return values to join them before it reaches this point.

Reply:
(whoa why did this comment thread just give me deja vu...)

If you go down to the very bottom of that page to check the first comment and the response following it, you see some discussion passing a list of values throuh the _same_ parameter.

I think @johngruen or @davismtl  wouldn't want a list of `something` showing up on SurveyGizmo as `something[0]` = `a`, `something[1]` = `b`, and `something[2]` = `a`, but rather `something` = `a,b,c`?

Reply:
Thanks.

Reply:
We also have at least one condition that returns an object 😢

Reply:
There is also a possibility of null being included in an array value (`reliersList: [ null, '9001' ]`), and because null is the desired value I assume this needs to be converted into an identifiable string; whether that's `"null"` or something else, I need direction.

Reply:
> I can't find any good authoritative SurveyGizmo documentation on how to pass a list of values, but from the comment section of an SurveyGizmo, I think it wants a comma separate values string. If that's the case, this won't work.
> 
> Maybe @johngruen can help on the format question here.

So, SG can do conditional filters on query strings, so i think a thing=a,b,c,d is fine here
<img width="283" alt="image" src="https://user-images.githubusercontent.com/3323249/86395878-444ee880-bca1-11ea-9649-ef6b74f2553f.png">

Reply:
Sounds good. What about the null values?

Reply:
for null, could we just drop that from the list?

Reply:
Perfect!